index.js

```diff
@@ -9,20 +9,23 @@ const client = new Discord.Client();
 let lastKristen = 0;
 
 client.on("ready", () => {
-	console.log(`Logged in as ${client.user.tag}!`);
+	console.log(util.format("Logged in as %s (id %d)", client.user.username, client.user.id));
+	setRandomActivity();
 });
 
 client.on("error", function(err) {
 	console.log(err);
 });
 
 client.on("message", (msg) => {
-	console.log(util.format("User %s (id: %d) wrote: %s", msg.author.username, msg.author.id, msg.content));
-
-	if (msg.author.username === client.user.username) {
+	// Ignore messages from ourselves and other bots to prevent spam
+	if (msg.author.username === client.user.username || msg.author.bot) {
```

Comment:
Can this be simplified? the `ourselves` here is the bot right? Can't we get away with just a 

```javascript
if (msg.author.bot)
```

Reply:
Its up to each bot author to tell Discord if they wanted to be treated as a "bot" and this done on the Discord developer portal, not within code. Its likely that msg.author.bot should be sufficient but I'm not convinced. Push back if you feel strongly about this, otherwise, I'm leaving it as is.

I will change the check from username to id as that seems more appropriate.

Reply:
I like your change to using id instead of username. Thanks for the info!